e2e/tests-dfx/assetscanister.bash

```diff
@@ -539,3 +539,19 @@ CHERRIES" "$stdout"
     assert_not_match "etag: my-etag"
     assert_match "etag: \"[a-z0-9]{64}\""
 }
+
+@test "asset configuration via .ic-assets.json5 - detect unused config" {
+    install_asset assetscanister
+
+    dfx_start
+
+    echo '[
+      {
+        "match": "nevermatchme",
+        "cache": { "max_age": 2000 }
+      }
+    ]' > src/e2e_project_frontend/assets/.ic-assets.json5
+
+    assert_command dfx deploy
+    assert_match "WARNING: 1 unmatched configuration in"
```

Comment:
Can you add an `assert_contains` here that shows what the `match` line would look like?

When I tried this, the output is as follows, which might not be expected:

```
WARNING: 1 unmatched configuration in /Users/ericswanson/w/dd/src/dd_frontend/assets/.ic-assets.json config file:
{
  "match": "/nevermatchme",
  "cache": {
    "max_age": 2000
  }
}
```

Reply:
thanks, added

Reply:
what do you think about the fact there is absolute path there `/Users/ericswanson/w/dd/src/dd_frontend/assets/.ic-assets.json` instead of project-relative path `/src/dd_frontend/assets/.ic-assets.json`?